src/chart/summaryTable/renderRow/makeBarChart.js

```diff
@@ -0,0 +1,152 @@
+    const chartSettings =
+        {x: {column: 'prop_n'
+            ,type: 'linear'
+            ,label: ''
+            ,format: '%'
+            ,domain: [0,null]}
+        ,y: {column: 'key'
+            ,type: 'ordinal'
+            ,label: ''}
+        ,marks:
+            [
+                {type: 'bar'
+                ,per: ['key']
+                ,summarizeX: 'mean'
+                ,tooltip: '[key]: [n] ([prop_n])'}
+            ]
+        ,gridlines: 'xy'
+        ,resizable: false
+        ,height: this_.height
+        ,margin: this_.margin
+        ,value_col: d.value_col
+        };
+
+    function onInit() {
+      //Add group labels.
+        if (this.config.group) {
+            const groupTitle = this.wrap
+                .append('p')
+                .text(`${this.config.group_col}: ${this.config.group} (n=${this.config.n})`);
+            this.wrap.node().parentNode
+                .insertBefore(groupTitle.node(), this.wrap.node())
+        }
+    }
+
+    function onResize() {
+      //Move y-axis labels to the right.
+        const ticks = this.wrap
+            .selectAll('g.y.axis g.tick');
+
+        ticks.select('text').remove();
+        ticks.append('title')
+            .text(d => d);
+        ticks
+            .append('text')
+            .attr(
+                {'text-anchor': 'start'
+                ,'alignment-baseline': 'middle'
+                ,'dx': '1em'
+                ,'x': this.plot_width})
+            .text(d => d.length < 30
+                ? d
+                : d.substring(0,30)+'...');
+
+      //Draw overall rates.
+        if (this.config.group) {
+            this.current_data
+                .forEach(d => {
+                    const overall = this.config.overall
+                        .filter(di => di.key === d.key)[0];
+
+                    if (overall) {
+                        const g = this.svg.append('g');
+                        const x = overall.prop_n;
+                        const y = overall.key;
+
+                      //Draw line representing the overall rate of the y-axis value.
+                        const rateLine = g
+                            .append('line')
+                            .attr(
+                                {'x1': this.x(x) + 1.5
+                                ,'y1': this.y(y)
+                                ,'x2': this.x(x) + 1.5
+                                ,'y2': this.y(y) + this.y.rangeBand()})
+                            .style(
+                                {'stroke': 'black'
+                                ,'stroke-width': '3px'
+                                ,'stroke-opacity': '1'});
+                        rateLine
+                            .append('title')
+                            .text(`Overall rate: ${d3.format('.1%')(x)}`);
+
+                      //Draw line from group rate to overall rate of the y-axis value.
+                        const diffLine = g
```

Comment:
I think I'd just make this visible when one of the relevant bars is hit with a mouseover.